services/blockchain-indexer/methods/dataService/network.js

```diff
@@ -13,12 +13,20 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const { getNetworkStatus } = require('./controllers/network');
+const {
+	getNetworkStatus,
+	getNetworkStatistics,
+} = require('./controllers/network');
 
 module.exports = [
 	{
 		name: 'network.status',
 		controller: getNetworkStatus,
 		params: {},
 	},
+	{
+		name: 'network.statistics',
+		controller: getNetworkStatistics,
+		params: {},
+	},
```

Comment:
I think controller for peers needs some cleanup as we define `network.statistics` here. We can remove `peers.statistics` completely from controller. We can discuss offline if needed.

Reply:
Applied 👍